content/en/api/integrations_webhook/code_snippets/result.webhooks_create.sh

```diff
@@ -1 +1,7 @@
-OK
\ No newline at end of file
+{
+    "name": <WEBHOOK_NAME>,
+    "url": <WEBHOOK_URL>,
+    "payload": null,
```

Comment:
I found it odd to show an example where the payload is null, when we have a default non-null.

Reply:
This is the behaviour right now, when people specify `null` they get `null` in the payload. We can change the documentation to show a custom payload which will look better 

We can implement showing a payload with the default payload as a response when `null` is specified (or when "payload" and "header" aren't specified in the request at all)